### PR TITLE
[cgal ] Update to 5.3.1

### PR DESCRIPTION
**Pull request by lrineau**
This pull-request updates the port `cgal` to the latest stable release CGAL-5.3.1.

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  All are supported. No need to update the CI baseline.

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  I read the guide, and the PR follows the rules, as far as I know.

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  Yes, commit https://github.com/microsoft/vcpkg/commit/584d5a161c54500a1318a347c98130ba8525379f.

### CODE REVIEW

File: ports/cgal/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "cgal",
-  "version": "5.3",
+  "version": "5.3.1",
+  "port-version": 1,
```
Comment (JonLiu1993) [inline ports/cgal/vcpkg.json:4]:
```suggestion
```

Comment (JonLiu1993) [reply ports/cgal/vcpkg.json:4]: When we update port to new version, we should remove "port-version"

File: ports/cgal/portfile.cmake [hunk 2]
```diff
@@ -13,18 +13,17 @@ vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
         qt WITH_CGAL_Qt5
 )
 
-vcpkg_configure_cmake(
+vcpkg_cmake_configure(
     SOURCE_PATH ${SOURCE_PATH}
```
Comment (JonLiu1993) [inline ports/cgal/portfile.cmake:17]:
```suggestion
    SOURCE_PATH "${SOURCE_PATH}"
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Comment by BillyONeal** [comment]
The CI failures look like this which suggests that something is unhappy with `find_package(CGAL)` for some customers.

```
[1/2] "D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/bin/cmake.exe" -S "D:/buildtrees/openmvs/src/v1.1.1-da3d990ca1.clean" -B .. "-DOpenMVS_USE_CUDA=OFF" "-DOpenMVS_USE_OPENMP=OFF" "-DOpenMVS_USE_NONFREE=ON" "-DOpenMVS_USE_CERES=OFF" "-DOpenMVS_USE_FAST_FLOAT2INT=ON" "-DOpenMVS_USE_FAST_INVSQRT=OFF" "-DOpenMVS_USE_FAST_CBRT=ON" "-DOpenMVS_USE_SSE=ON" "-DOpenMVS_USE_OPENGL=ON" "-DOpenMVS_USE_BREAKPAD=OFF" "-DCMAKE_MAKE_PROGRAM=D:/downloads/tools/ninja/1.10.2-windows/ninja.exe" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/a/1/s/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x86-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/a/1/s/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x86" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/a/1/s" "-D_VCPKG_INSTALLED_DIR=D:/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DCMAKE_DISABLE_SOURCE_CHANGES=ON" "-DOpenMVS_BUILD_TOOLS=ON" -G "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=D:/packages/openmvs_x86-windows"
FAILED: ../CMakeCache.txt 
"D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/bin/cmake.exe" -S "D:/buildtrees/openmvs/src/v1.1.1-da3d990ca1.clean" -B .. "-DOpenMVS_USE_CUDA=OFF" "-DOpenMVS_USE_OPENMP=OFF" "-DOpenMVS_USE_NONFREE=ON" "-DOpenMVS_USE_CERES=OFF" "-DOpenMVS_USE_FAST_FLOAT2INT=ON" "-DOpenMVS_USE_FAST_INVSQRT=OFF" "-DOpenMVS_USE_FAST_CBRT=ON" "-DOpenMVS_USE_SSE=ON" "-DOpenMVS_USE_OPENGL=ON" "-DOpenMVS_USE_BREAKPAD=OFF" "-DCMAKE_MAKE_PROGRAM=D:/downloads/tools/ninja/1.10.2-windows/ninja.exe" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/a/1/s/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x86-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/a/1/s/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x86" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/a/1/s" "-D_VCPKG_INSTALLED_DIR=D:/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DCMAKE_DISABLE_SOURCE_CHANGES=ON" "-DOpenMVS_BUILD_TOOLS=ON" -G "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=D:/packages/openmvs_x86-windows"
-- The C compiler identification is MSVC 19.29.30138.0
-- The CXX compiler identification is MSVC 19.29.30138.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.29.30133/bin/Hostx64/x86/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.29.30133/bin/Hostx64/x86/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
Compiling with C++14
-- Looking for inttypes.h
-- Looking for inttypes.h - found
-- Found OpenGL: opengl32   
CMake Warning at D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindBoost.cmake:1354 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindBoost.cmake:1477 (_Boost_COMPONENT_DEPENDENCIES)
  D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindBoost.cmake:2087 (_Boost_MISSING_DEPENDENCIES)
  D:/installed/x86-windows/share/boost/vcpkg-cmake-wrapper.cmake:11 (_find_package)
  C:/a/1/s/scripts/buildsystems/vcpkg.cmake:742 (include)
  CMakeLists.txt:123 (FIND_PACKAGE)


CMake Warning at D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindBoost.cmake:1354 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindBoost.cmake:1477 (_Boost_COMPONENT_DEPENDENCIES)
  D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindBoost.cmake:2087 (_Boost_MISSING_DEPENDENCIES)
  D:/installed/x86-windows/share/boost/vcpkg-cmake-wrapper.cmake:11 (_find_package)
  C:/a/1/s/scripts/buildsystems/vcpkg.cmake:742 (include)
  CMakeLists.txt:123 (FIND_PACKAGE)


CMake Warning at D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindBoost.cmake:1354 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindBoost.cmake:1477 (_Boost_COMPONENT_DEPENDENCIES)
  D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindBoost.cmake:2087 (_Boost_MISSING_DEPENDENCIES)
  D:/installed/x86-windows/share/boost/vcpkg-cmake-wrapper.cmake:11 (_find_package)
  C:/a/1/s/scripts/buildsystems/vcpkg.cmake:742 (include)
  CMakeLists.txt:123 (FIND_PACKAGE)


CMake Warning at D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindBoost.cmake:1354 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindBoost.cmake:1477 (_Boost_COMPONENT_DEPENDENCIES)
  D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindBoost.cmake:2087 (_Boost_MISSING_DEPENDENCIES)
  D:/installed/x86-windows/share/boost/vcpkg-cmake-wrapper.cmake:11 (_find_package)
  C:/a/1/s/scripts/buildsystems/vcpkg.cmake:742 (include)
  CMakeLists.txt:123 (FIND_PACKAGE)


CMake Warning at D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindBoost.cmake:1354 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindBoost.cmake:1477 (_Boost_COMPONENT_DEPENDENCIES)
  D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindBoost.cmake:2087 (_Boost_MISSING_DEPENDENCIES)
  D:/installed/x86-windows/share/boost/vcpkg-cmake-wrapper.cmake:11 (_find_package)
  C:/a/1/s/scripts/buildsystems/vcpkg.cmake:742 (include)
  CMakeLists.txt:123 (FIND_PACKAGE)


-- Found Boost: D:/installed/x86-windows/include (found version "1.77.0") found components: iostreams program_options system serialization regex 
-- Found OpenCV: D:/installed/x86-windows (found version "4.5.4") 
-- OpenCV 4.5.4 found (include: D:/installed/x86-windows/include)
CMake Error at D:/installed/x86-windows/share/cgal/CGALConfig-installation-dirs.cmake:1:
  Parse error.  Expected a newline, got identifier with text "set".
Call Stack (most recent call first):
  D:/installed/x86-windows/share/cgal/CGALConfig.cmake:44 (include)
  C:/a/1/s/scripts/buildsystems/vcpkg.cmake:788 (_find_package)
  libs/MVS/CMakeLists.txt:2 (FIND_PACKAGE)


CMake Error at C:/a/1/s/scripts/buildsystems/vcpkg.cmake:788 (_find_package):
  Found package configuration file:

    D:/installed/x86-windows/share/cgal/CGALConfig.cmake

  but it set CGAL_FOUND to FALSE so package "CGAL" is considered to be NOT
  FOUND.
Call Stack (most recent call first):
  libs/MVS/CMakeLists.txt:2 (FIND_PACKAGE)


-- Configuring incomplete, errors occurred!
See also "D:/buildtrees/openmvs/x86-windows-rel/CMakeFiles/CMakeOutput.log".
[2/2] "D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/bin/cmake.exe" -S "D:/buildtrees/openmvs/src/v1.1.1-da3d990ca1.clean" -B "../../x86-windows-dbg" "-DOpenMVS_USE_CUDA=OFF" "-DOpenMVS_USE_OPENMP=OFF" "-DOpenMVS_USE_NONFREE=ON" "-DOpenMVS_USE_CERES=OFF" "-DOpenMVS_USE_FAST_FLOAT2INT=ON" "-DOpenMVS_USE_FAST_INVSQRT=OFF" "-DOpenMVS_USE_FAST_CBRT=ON" "-DOpenMVS_USE_SSE=ON" "-DOpenMVS_USE_OPENGL=ON" "-DOpenMVS_USE_BREAKPAD=OFF" "-DCMAKE_MAKE_PROGRAM=D:/downloads/tools/ninja/1.10.2-windows/ninja.exe" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/a/1/s/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x86-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/a/1/s/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x86" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/a/1/s" "-D_VCPKG_INSTALLED_DIR=D:/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DCMAKE_DISABLE_SOURCE_CHANGES=ON" "-DOpenMVS_BUILD_TOOLS=OFF" -G "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=D:/packages/openmvs_x86-windows/debug"
FAILED: ../../x86-windows-dbg/CMakeCache.txt 
"D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/bin/cmake.exe" -S "D:/buildtrees/openmvs/src/v1.1.1-da3d990ca1.clean" -B "../../x86-windows-dbg" "-DOpenMVS_USE_CUDA=OFF" "-DOpenMVS_USE_OPENMP=OFF" "-DOpenMVS_USE_NONFREE=ON" "-DOpenMVS_USE_CERES=OFF" "-DOpenMVS_USE_FAST_FLOAT2INT=ON" "-DOpenMVS_USE_FAST_INVSQRT=OFF" "-DOpenMVS_USE_FAST_CBRT=ON" "-DOpenMVS_USE_SSE=ON" "-DOpenMVS_USE_OPENGL=ON" "-DOpenMVS_USE_BREAKPAD=OFF" "-DCMAKE_MAKE_PROGRAM=D:/downloads/tools/ninja/1.10.2-windows/ninja.exe" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/a/1/s/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x86-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/a/1/s/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x86" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/a/1/s" "-D_VCPKG_INSTALLED_DIR=D:/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DCMAKE_DISABLE_SOURCE_CHANGES=ON" "-DOpenMVS_BUILD_TOOLS=OFF" -G "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=D:/packages/openmvs_x86-windows/debug"
-- The C compiler identification is MSVC 19.29.30138.0
-- The CXX compiler identification is MSVC 19.29.30138.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.29.30133/bin/Hostx64/x86/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.29.30133/bin/Hostx64/x86/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
Compiling with C++14
-- Looking for inttypes.h
-- Looking for inttypes.h - found
-- Found OpenGL: opengl32   
CMake Warning at D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindBoost.cmake:1354 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindBoost.cmake:1477 (_Boost_COMPONENT_DEPENDENCIES)
  D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindBoost.cmake:2087 (_Boost_MISSING_DEPENDENCIES)
  D:/installed/x86-windows/share/boost/vcpkg-cmake-wrapper.cmake:11 (_find_package)
  C:/a/1/s/scripts/buildsystems/vcpkg.cmake:742 (include)
  CMakeLists.txt:123 (FIND_PACKAGE)


CMake Warning at D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindBoost.cmake:1354 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindBoost.cmake:1477 (_Boost_COMPONENT_DEPENDENCIES)
  D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindBoost.cmake:2087 (_Boost_MISSING_DEPENDENCIES)
  D:/installed/x86-windows/share/boost/vcpkg-cmake-wrapper.cmake:11 (_find_package)
  C:/a/1/s/scripts/buildsystems/vcpkg.cmake:742 (include)
  CMakeLists.txt:123 (FIND_PACKAGE)


CMake Warning at D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindBoost.cmake:1354 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindBoost.cmake:1477 (_Boost_COMPONENT_DEPENDENCIES)
  D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindBoost.cmake:2087 (_Boost_MISSING_DEPENDENCIES)
  D:/installed/x86-windows/share/boost/vcpkg-cmake-wrapper.cmake:11 (_find_package)
  C:/a/1/s/scripts/buildsystems/vcpkg.cmake:742 (include)
  CMakeLists.txt:123 (FIND_PACKAGE)


CMake Warning at D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindBoost.cmake:1354 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindBoost.cmake:1477 (_Boost_COMPONENT_DEPENDENCIES)
  D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindBoost.cmake:2087 (_Boost_MISSING_DEPENDENCIES)
  D:/installed/x86-windows/share/boost/vcpkg-cmake-wrapper.cmake:11 (_find_package)
  C:/a/1/s/scripts/buildsystems/vcpkg.cmake:742 (include)
  CMakeLists.txt:123 (FIND_PACKAGE)


CMake Warning at D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindBoost.cmake:1354 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindBoost.cmake:1477 (_Boost_COMPONENT_DEPENDENCIES)
  D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/share/cmake-3.21/Modules/FindBoost.cmake:2087 (_Boost_MISSING_DEPENDENCIES)
  D:/installed/x86-windows/share/boost/vcpkg-cmake-wrapper.cmake:11 (_find_package)
  C:/a/1/s/scripts/buildsystems/vcpkg.cmake:742 (include)
  CMakeLists.txt:123 (FIND_PACKAGE)


-- Found Boost: D:/installed/x86-windows/include (found version "1.77.0") found components: iostreams program_options system serialization regex 
-- Found OpenCV: D:/installed/x86-windows (found version "4.5.4") 
-- OpenCV 4.5.4 found (include: D:/installed/x86-windows/include)
CMake Error at D:/installed/x86-windows/share/cgal/CGALConfig-installation-dirs.cmake:1:
  Parse error.  Expected a newline, got identifier with text "set".
Call Stack (most recent call first):
  D:/installed/x86-windows/share/cgal/CGALConfig.cmake:44 (include)
  C:/a/1/s/scripts/buildsystems/vcpkg.cmake:788 (_find_package)
  libs/MVS/CMakeLists.txt:2 (FIND_PACKAGE)


CMake Error at C:/a/1/s/scripts/buildsystems/vcpkg.cmake:788 (_find_package):
  Found package configuration file:

    D:/installed/x86-windows/share/cgal/CGALConfig.cmake

  but it set CGAL_FOUND to FALSE so package "CGAL" is considered to be NOT
  FOUND.
Call Stack (most recent call first):
  libs/MVS/CMakeLists.txt:2 (FIND_PACKAGE)


-- Configuring incomplete, errors occurred!
See also "D:/buildtrees/openmvs/x86-windows-dbg/CMakeFiles/CMakeOutput.log".
ninja: build stopped: subcommand failed.

```

**Comment by lrineau** [comment]
Hi @BillyONeal, the file `CGALConfig.cmake` (actually one of its companions, `CGALConfig-installation-dirs.cmake`) is a victim of the bug of `vcpkg_cmake_config_fixup()` fixed by #22236. I am waiting for #22236 to be merged, so that I can continue with the update of the `cgal` port.

**Comment by BillyONeal** [comment]
Thanks for the update!

**Pull request merged**